### ISSUE TITLE
Monthly Chat Agenda August (2020-08-03)

### ISSUE THREAD

**Issue by dune73** [coreruleset/coreruleset#1853]
This is the Agenda for the Monthly CRS Chat.

The chat is going to happen on https://owasp.slack.com in the channel #coreruleset on Monday, August 3rd, at 20:30 CET.

Items on the Agenda:  (see previous meetings decisions: [here](https://github.com/coreruleset/coreruleset/issues/1836))

## PRs

* #1848- Permit "version" string in content-type header
* #1847 - Suppress rule 200002 when Nextcloud mobile app accesses shares through API
* ~~#1846 - feat(ftw): use master to get py3 support~~ (still working through the errors - fgsch)
* #1845 - Stop decoding things twice (= revert #578)
* #1843 - Added new nextcloud excludes for the "Security & setup warnings" test and session tokens
* #1842 - Drop unneeded escapes
* #1839 - Update REQUEST-920-PROTOCOL-ENFORCEMENT.conf
* #1817 - Fix false positives with rule 942360
* #1794 - Fix bypass 942190
* #1793 - SQLi using set var at PL1

## Other items

* Idea to tighten up the monthly project meeting by removing the issues from the agenda and running a separate issue chat once every month; like the 2nd or 3rd Monday or something. We did it that way in July 2020, but the turnout was very low.
* OWASP Donation policy (and other policies)

Feel free to add items as you see fit either above, or below as comments.

## Open Issues

In January 2020, we decided to look into 10 issues at the chat every month. But only after the Other items. Pick the issues before the meeting and list them 
* Issue slot 1: #1819 - request to remove the `ctl:auditLogParts=+E` actions
* Issue slot 2: #1827 - FP at rule 951120 - no user feedback
* Issue slot 3: #1857 - FPs in 942420 and 942421
* Issue slot 4: #1850 - FPs in 942350 and 932130
* Issue slot 5: #1849 - Problems with workaround proposed by @spartantri 
* Issue slot 6: #1838 - Nextcloud FP
* Issue slot 7: #1835 - FP on 932200
* Issue slot 8: #1834 - FP on 932130 (WordPress)
* Issue slot 9: #1811 - FP on 920420
* Issue slot 10: #1768 - FP on 932120

If you are not yet on the OWASP Slack, here is your invite: https://owasp-slack.herokuapp.com/ .
Everybody is welcome to join our community chat.

**Comment by dune73** [comment]
## Decision

#1848 - Assign to @lifeforms, will ask for test, then merge
#1847 - Assign to @fzipitria, who will guide the contributors working on this, then merge
#1845 - Assign to @franbuehler for review
#1843 - Assign to @fzipitria, who will guide the contributor working on this, then merge
#1842 - Merge
#1839 - Assign to @lifeforms to ask for a test, hope the lint error goes away and then merge
#1817 - Give it another month, added the "needs action" label
#1794 - Work in progress, @theMiddle plans to fix it over the weekend
#1793 - Work in progress, @theMiddle will try and fix open issues, shift to PL2 and then have it merged.

## Other items

* Initiate communication with packagers etc. (Assigned to @dune73: Make a communication plan)
* New issue meeting on 3rd Monday in August and September. We'll reassess the situation in October then.
* Donation Policy: Spread the word, ask OWASP contacts to sign.
* Demo site by @csanders is almost done!
